frontend/src/advising/ManageStudents/StudentView.tsx

```diff
@@ -0,0 +1,226 @@
+import { NonEditableSchedule } from "../../components/Schedule/ScheduleComponents";
+import { SwitchPlanList } from "../../components/SwitchPlan/SwitchPlanList";
+import { PlanTitle, ButtonHeader, ScheduleWrapper, Container } from "./Shared";
+import { useHistory, useParams } from "react-router";
+
+const StudentViewContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  > * {
+    border: 1px solid red;
+    border-radius: 10px;
+    height: 70vh;
+    padding: 30px;
+  }
+  * {
+    font-family: Roboto;
+    font-style: normal;
+  }
+`;
+
+const SchedulePreviewContainer = styled.div`
+  overflow: hidden;
+  flex: 5;
+`;
+
+const StudentInfoContainer = styled.div`
+  flex: 1;
+  margin-right: 20px;
+`;
+
+const NoPlanContainer = styled.div`
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StudentInfoDisplay = styled.div`
+  height: 100%;
+`;
+
+const AvatarWrapper = styled.div`
+  height: 25%;
+  > * {
+    background-color: #d3898d !important;
+    color: #fff !important;
+    font-size: 5vw !important;
+    margin: 0% 10% !important;
+    width: 80% !important;
+    height: 100% !important;
+  }
+`;
+
+const StudentInfoTextWrapper = styled.div`
+  height: 25%;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  > * {
+  }
+`;
+const Text = styled.div`
+  font-size: 12px;
+  font-weight: normal;
+  line-height: 20px;
+`;
+const NameText = styled.div`
+  line-height: 28px;
+  text-align: center;
+  font-size: 14px;
+  font-weight: 500;
+`;
+const TitleText = styled.div`
+  font-size: 24px;
+  line-height: 20px;
+  font-weight: 500;
+`;
+const PlanText = styled.div`
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 28px;
+`;
+const PlanListContainer = styled.div`
+  width: 100%;
+  height: 35%;
+  overflow: hidden;
+  &:hover {
+    overflow-y: auto;
+  }
+`;
+const ButtonContainer = styled.div`
+  margin: 5% 0%;
+  justify-content: center;
+  display: flex;
+`;
+
+interface ParamProps {
+  id: string;
+}
+
+export const StudentView: React.FC = () => {
+  const history = useHistory();
+  const routeParams = useParams<ParamProps>();
+  const id = Number(routeParams.id);
+  const [fetchingStudent, setFetchingStudent] = useState(true);
+  const [student, setStudent] = useState<IUserData | null>(null);
+  const [noPlans, setNoPlans] = useState(false);
+
+  const dispatch = useDispatch();
+  const { planName, planId } = useSelector((state: AppState) => ({
+    planName: getActivePlanNameFromState(state),
+    planId: safelyGetActivePlanIdFromState(state),
+  }));
+
+  useEffect(() => {
+    fetchUser(id).then(response => {
+      setStudent(response.user);
+      setFetchingStudent(false);
+    });
+    findAllPlansForUser(id).then((plans: IPlanData[]) => {
+      dispatch(setUserPlansAction(plans, 2020));
+      if (!plans || !plans.length || !plans[0].schedule) setNoPlans(true);
+    });
+  }, []);
+
+  return (
+    <Container>
```

Comment:
nit but can we break up this JSX? this render function is pretty big, could we split it into smaller components? For example, we could have the StudentPreviewContainer and SchedulePreviewContainer as different components